be/src/exec/schema_scanner/schema_workload_groups_scanner.cpp

```diff
@@ -31,13 +31,13 @@ namespace doris {
 std::vector<SchemaScanner::ColumnDesc> SchemaWorkloadGroupsScanner::_s_tbls_columns = {
         {"ID", TYPE_BIGINT, sizeof(int64_t), true},
         {"NAME", TYPE_VARCHAR, sizeof(StringRef), true},
-        {"CPU_SHARE", TYPE_BIGINT, sizeof(int64_t), true},
-        {"MEMORY_LIMIT", TYPE_VARCHAR, sizeof(StringRef), true},
-        {"ENABLE_MEMORY_OVERCOMMIT", TYPE_VARCHAR, sizeof(StringRef), true},
+        {"MIN_CPU_PERCENT", TYPE_BIGINT, sizeof(int64_t), true},
+        {"MAX_CPU_PERCENT", TYPE_BIGINT, sizeof(int64_t), true},
+        {"MIN_MEMORY_PERCENT", TYPE_BIGINT, sizeof(int64_t), true},
+        {"MAX_MEMORY_PERCENT", TYPE_BIGINT, sizeof(int64_t), true},
```

Comment:
In an 1024G memoy's node, 1% means 10G memory, 0.5% means 5G memory, it's  not a small waste , maybe double is better.

Reply:
I think it is ok. If the memory is very large, use do not need care about 5GB memory. 
But this is a bug, the type should be varchar, not int64.